src/cmd/tools/m3db_tool/main/common/http.go

```diff
@@ -0,0 +1,115 @@
+package common
+
+import (
+	"flag"
+	"fmt"
+	"go.uber.org/zap"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+var (
+	EndPoint *string
+)
+
+// This is used across all the commands
+func init() {
+	EndPoint = flag.String("endpoint", "http://localhost:7201", "The url for target m3db backend.")
+}
+
+func DoGet(url string, logger *zap.SugaredLogger, getter func(reader io.Reader, logger *zap.SugaredLogger)) {
+
+	logger.Debugf("DoGet:url:%s:\n", url)
+
+	resp, err := http.Get(url)
```

Comment:
I think that you need to use `http.Client` with a timeout instead of `http.Get`

Reply:
I'm curious why you're saying I need to do that. Its working. Is it because the default timeout is bad or something?

Reply:
The default doesn't have a timeout. If you leave it like this, then in case of network issues it might hang forever. Ideally, you should always use timeouts except for demo purposes.

Reply:
https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779